src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -173,27 +173,30 @@ internal async Task<IReadOnlyCollection<ResourceSearchParameterStatus>> GetSearc
 
         internal async Task ApplySearchParameterStatus(IReadOnlyCollection<ResourceSearchParameterStatus> updatedSearchParameterStatus, CancellationToken cancellationToken)
         {
-            var updated = new List<SearchParameterInfo>();
-
-            foreach (var paramStatus in updatedSearchParameterStatus)
+            if (updatedSearchParameterStatus.Any())
```

Comment:
>if (updatedSearchParameterStatus.Any() [](http://example.com/codeflow?start=12&length=38)

super nit:
I personally find 
```
if (!updatedSearchParameterStatus.Any())
[
 return Task.CompletedTask;
}
```
more clean than wrapping whole method in if statement.